lib/services/http-service.js

```diff
@@ -220,7 +220,7 @@ function httpServiceFactory(
         );
 
         //re-route common and current
-        var versionPath = configuration.get('versionBase') || '1.1';
+        var versionPath = configuration.get('versionBase') || '2.0';
```

Comment:
Also my recommendation, this is the common way we handle the default value before.